examples/kind/konnectivity-server.yaml

```diff
@@ -53,6 +53,17 @@ spec:
       nodeSelector:
         node-role.kubernetes.io/control-plane: ""
       hostNetwork: true
+      initContainers:
+      - name: busybox
+        image: busybox
+        command:
+        - sh
+        - -c
+        - |
+          chmod -R 777 /etc/kubernetes
```

Comment:
774? 744? 444?

Reply:
I have tested it with 444 446 and 447, only working with 447, This is a bit counter-intuitive since the ANP server only requires read permissions (except /etc/kubernetes/konnectivity-server ).

How about change this directory to `/etc/srv/kubernetes` ? To keep it consistent with `examples/kubernetes/konnectivity-server.yaml`, but the apiserver key still world-readable on `/etc/srv/kubernetes`  :(

Reply:
Does manually creating the required files and starting ANP with the 1002 user increase the confidence in merging the PR?

```shell
...
securityContext:
        runAsUser: 1002
        runAsGroup: 1000
...
```